history/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/impl/HistoryCleanupJob.java

```diff
@@ -0,0 +1,337 @@
+package pro.taskana.simplehistory.impl;
```

Comment:
All other jobs reside in a package "jobs". Maybe we should stick to this structure for the history as well?